File: minMax.c
#include<stdio.h>
#define N 10

int main()
{
    int a[N];
    int i;
    //int min=a[0],max=a[0];
    printf("Enter the elements of array :\n");
    for(i=0; i<N; i++)
    {
        scanf("%d",&a[i]);
    }
    int min=a[0],max=a[0];
    for(i=1; i<N-1; i++)
    {
        if(min>a[i])
        {
            min=a[i];
        }
        if(max<a[i])
        {
            max=a[i];
        }
    }
    printf("Min  :%d\n",min);
    printf("Max  :%d",max);
    return 0;

}

File: Min_Max.java
//12. Find the largest and smallest elements in an array.
class MinMax {
    public static void main (String[] args){
        int[] arr = {3, 6, 54, 56, 34, 556, 87, 95};
        int min = arr[0];
        int max = arr[0];
        for(int i = 0; i < arr.length; i++) {
            if(arr[i] > max) {
                max = arr[i];
            }
            if(arr[i] < min) {
                min = arr[i];
            }
        }
        System.out.println("Minimum = " + min + " Maximum = " + max);
    }
}